ui/app/templates/permissions_modal.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
         <form role="form" ng-submit="grantRole()">
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1342231 for the real error, https://bugzilla.mozilla.org/show_bug.cgi?id=1342234 for the fact that the error wasn't shown.